test/docsExamples.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
File name: Perhaps add "style" to the name, or do you intend this file to test more at a later point in time?

Reply:
Yea I thought the same, this way it sounds like it's actually testing the examples for compilation and not only style.